setup.cfg

```diff
@@ -45,7 +45,7 @@ install_requires =
     spherical-geometry>=1.2.20
     stcal>=0.4.1
     stdatamodels>=0.3.0,<1.0
-    stpipe>=0.1.0,<1.0
+    stpipe>=0.3.1,<1.0
```

Comment:
Should this be here?

Reply:
The latest (or nearly latest) version of stpipe is required in order to allow for proper parsing of arguments that contain a list of values (which is what `center_xy` needs).